### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Sameony**
Added myCart page
image

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
Looks perfect, no changes required.

File: Styles/cart.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  background-color: var(--light-caution);
+}
+.cartImg {
+  overflow: hidden;
+}
+.cartActionButtons {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: space-around;
```
Comment (krishanu2911) [inline Styles/cart.css]: Just a small suggestion that you can move this flex property to util.

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go no changes are required just added a small suggestion.

File: Styles/cart.css (hunk 1 above)
Comment (Sameony) [reply Styles/cart.css]: thanks for pointing out

**Pull request merged**